pyls/plugins/preload_imports.py

```diff
@@ -30,5 +30,7 @@ def pyls_initialize(config):
         try:
             __import__(mod_name)
             log.debug("Preloaded module %s", mod_name)
-        except ImportError:
+        except (ImportError, ValueError):
```

Comment:
What do you think about changing this to `except Exception`? In my experience modules can fail to be imported for a variety of reasons. So I think it's better to be on the safe side, especially because this causes the server to crash at initialization.